### PR TITLE
send accepted org invite notif to inviter

### PR DESCRIPTION
**Pull request by camhashemi**
@leogrim @ryanpbrewster seems like this was removed in a refactor, not sure if it was intentional or not.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/UserInteractionCommander.scala [hunk 1]
```diff
@@ -81,5 +81,5 @@ object UserInteraction {
 }
 
 sealed trait InteractionRecipient
-case class UserRecipient(id: Id[User]) extends InteractionRecipient
-case class EmailRecipient(address: EmailAddress) extends InteractionRecipient
+case class UserInteractionRecipient(id: Id[User]) extends InteractionRecipient
+case class EmailInteractionRecipient(address: EmailAddress) extends InteractionRecipient
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/UserInteractionCommander.scala:85]: these have naming collisions with subclasses of `keepit.notify.model.Recipient`

Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/UserInteractionCommander.scala:85]: :+1:

**Comment by leogrim** [comment]
lgtm!

**Comment by ryanpbrewster** [comment]
:+1: merge away

**Pull request merged**